packages/widgets/src/components/Pagination/PaginationButtons.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react';
+import { useTheme } from '@emotion/react';
+import { Button } from '@ssa-ui-kit/core';
+
+import { pageBtnStyles, selectedPageBtnStyles } from './styles';
+import { IPaginationButtonsProps, IPageButtonProps } from './types';
+
+const Break = () => <span css={{ cursor: 'default' }}>...</span>;
+
+const PageButton = ({ onClick, isSelected, page }: IPageButtonProps) => {
+  const theme = useTheme();
+  const styles = useMemo(() => {
+    return isSelected ? selectedPageBtnStyles(theme) : pageBtnStyles(theme);
+  }, [isSelected]);
+
+  return (
+    <Button
+      size="small"
+      variant="secondary"
+      onClick={
+        isSelected
+          ? () => {
+              /*no-op*/
+            }
+          : onClick
+      }
+      css={styles}
+      aria-label={isSelected ? `Current page ${page}` : `Go to page ${page}`}
+      aria-current={isSelected}>
+      {page}
+    </Button>
+  );
+};
+
+export const PaginationButtons = ({
+  range,
+  selectedPage,
+  onClick,
+}: IPaginationButtonsProps) => {
+  return range.map((page, index) => {
```

Comment:
I think, we need additional checking for the range

Reply:
Ok, will do.